srcpkgs/libgnt/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'libgnt'
+pkgname=libgnt
+version=2.14.2
+revision=1
+build_style=meson
+build_helper=gir
+hostmakedepends="pkg-config glib-devel gtk-doc"
+makedepends="libxml2-devel ncurses-devel python3-devel glib-devel gtk-doc"
+short_desc="GLib Ncurses Toolkit"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="GPL-2.0-or-later"
+homepage="https://keep.imfreedom.org/libgnt/"
+distfiles="${SOURCEFORGE_SITE}/pidgin/libgnt/${version}/libgnt-${version}.tar.xz"
+checksum=61cf74b14eef10868b2d892e975aa78614f094c8f4d30dfd1aaedf52e6120e75
+patch_args=" -Np1"
+
+pre_configure() {
+	# disable doc for cross builds
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i "/subdir('doc')/d" meson.build
+	fi
+}
+
+libgnt-devel_package() {
+	depends="libglib-devel libgnt>=${version}_${revision}"
+	short_desc="IM library extracted from Pidgin (GNT) - development files"
```

Comment:
```suggestion
	short_desc+=" - development files"
```

But maybe fix the main `short_desc` to match whatever you prefer?

Reply:
oops